lnbits/extensions/lnaddress/lnurl.py

```diff
@@ -1,3 +1,4 @@
+import hashlib
```

Comment:
is this file related to the PR? lnbits/extensions/lnaddress/lnurl.py

Reply:
> Can we get that privatekey auto-generated and stored in db, so its different for each install?

Yes, it's important that each installation gets it's own keypair! It already works this way, if you don't check-in the data files. ;-)

Sure, this can be put in db. Just used file system because the lib works this way. Privkey alone should do it.